redaxo/src/core/lib/fragment.php

```diff
@@ -1,6 +1,13 @@
 <?php
 
 /**
+ * Methods declared here are available to be called from within a fragment-file, even if not public:
+ *
+ * @method void subfragment(string $filename, array $params = [])
+ * @method string getSubfragment(string $filename, array $params = [])
+ * @method string i18n(string $key, ...$replacements)
+ * @method mixed escape($value, $strategy = 'html')
+ *
```

Comment:
alle methoden die es innnerhalb von fragmenten gibt, hier via PHPDoc publiziert damit wir sie weiterhin mit `protected` visibility deklariert lassen können

Reply:
funktioniert in phpstan, aber noch nicht in psalm:

upstream issue https://github.com/vimeo/psalm/issues/4568

Reply:
für psalm jetzt `@psalm-scope-this rex_fragment` ergänzt